### ISSUE TITLE
CVPR 2019 Reading List

### ISSUE THREAD

**Issue by jodoin** [vitalab/vitalab.github.io#386]
### MUST READ
# Deep learning
- [ ] Bag of Tricks for Image Classification with Convolutional Neural Networks
- [ ] Selective Kernel Networks
- [ ] Stand-Alone Self-Attention in Vision Models (arxiv.org/pdf/1906.05909.pdf)
- [ ] Kervolutional Neural Networks
- [x] Why ReLU Networks Yield High-Confidence Predictions Far Away From the Training Data and How to Mitigate the Problem
- [ ] SDC – Stacked Dilated Convolution: A Unified Descriptor Network for Dense Matching Tasks
- [x] A General and Adaptive Robust Loss Function
- [x] Attention Branch Network: Learning of Attention Mechanism for Visual Explanation
- [ ] RePr: Improved Training of Convolutional Filters

# Domain Adaptation and Multitask learning
- [x] Towards Universal Object Detection by Domain Attention (*Charles*)
- [x] d-SNE: Domain Adaptation Using Stochastic Neighborhood Embedding
- [ ] Contrastive Adaptation Network for Unsupervised Domain Adaptation
- [ ] Learning without memorizing
- [ ] Efficient Multi-Domain Learning by Covariance Normalization
- [x] CrDoCo: Pixel-Level Domain Transfer With Cross-Domain Consistency
- [ ] All About Structure: Adapting Structural Information Across Domains for Boosting Semantic Segmentation
- [x] End-To-End Multi-Task Learning With Attention
- [ ] Attentive Single-Tasking of Multiple Tasks
- [ ] ADVENT: Adversarial Entropy Minimization for Domain Adaptation in Semantic Segmentation
- [ ] Unsupervised Visual Domain Adaptation: A Deep Max-Margin Gaussian Process Approach

# Remote sensing
- [ ] DARNet: Deep Active Ray Network for Building Segmentation
- [ ] Late or Earlier Information Fusion from Depth and Spectral Data? Large-Scale Digital Surface Model Refinement by Hybrid-cGAN

# Architecture search and pruning
- [ ] Centripetal SGD for Pruning Very Deep Convolutional Networks With Complicated Structure
- [ ] Auto-DeepLab: Hierarchical Neural Architecture Search for Semantic Image Segmentation
- [ ] Filter Pruning via Geometric Median for Deep Convolutional Neural Networks Acceleration

# Segmentation
- [ ] Hybrid Task Cascade for Instance Segmentation
- [x] Graph-Based Global Reasoning Networks
- [ ] Panoptic Feature Pyramid Networks

# Weekly supervised learning - Adaptive learning and stuff
- [ ] Label Propagation for Deep Semi-Supervised Learning
- [x] Iterative Projection and Matching: Finding Structure-Preserving Representatives and Its Application to Computer Vision
- [x] Learning Loss for Active Learning

### SUGGESTED
- [x] Hardness-Aware Deep Metric Learning
- [ ] Large-scale interactive object segmentation with human annotators
- [ ] Content-Aware Multi-Level Guidance for Interactive Instance Segmentation
- [ ] Probabilistic End-To-End Noise Correction for Learning With Noisy Labels
- [ ] Deeply-Supervised Knowledge Synergy
- [ ] Fast Interactive Object Annotation With Curve-GCN
- [ ] Attention-Aware Multi-Stroke Style Transfer
- [ ] Learning Semantic Segmentation From Synthetic Data: A Geometrically Guided Input-Output Adaptation Approach
- [ ]  Striking the Right Balance With Uncertainty
- [ ]  Large-Scale Long-Tailed Recognition in an Open World
- [ ] AET vs. AED: Unsupervised Representation Learning by Auto-Encoding Transformations Rather Than Data
- [ ] Mitigating Information Leakage in Image Representations: A Maximum Entropy Approach

### Carl's Picks

- [x] deep virtual networks for memory efficient inference of multiple tasks
